libmate-desktop/mate-colorsel.c

```diff
@@ -2826,9 +2818,9 @@ mate_color_selection_palette_to_string (const GdkColor *colors,
 
       strs[i] =
         g_strdup_printf ("#%2X%2X%2X",
-                         colors[i].red / 256,
-                         colors[i].green / 256,
-                         colors[i].blue / 256);
+                         (unsigned int)colors[i].red / 256,
+                         (unsigned int)colors[i].green / 256,
+                         (unsigned int)colors[i].blue / 256);
```

Comment:
Is this correct?  I would have expected to have to do `colors[i].red * 255` given `GdkRGBA` uses `0..1` ranges (and ofc same for the other channels).

Reply:
done